nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    predictions = nf.predict(df=pred_df, futr_df=futr_df, static_df=static).reset_index()
</pre><p>With the id as col env variable the reset_index won't be necessary anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>